### PR TITLE
Add About page

### PR DESCRIPTION
**Pull request by rawrmonstar**
- Add View showcase link on homepage (in `<Companies />`)
- Add [`anchorate`](https://github.com/adjohnson916/anchorate) as a dependency to handle scrolling with hash links
- Create About screen, `<Showcase />`, and `<ShowcaseApp />`

Addresses #92 and #94 

Hi @ebrillhart! Can you take a look at this? I think all that needs to be done is to replace the copy and add any extra styling it might need.

### CODE REVIEW

**Comment by ebrillhart** [comment]
thanks, @rawrmonstar! going to loop @paulathevalley in on this too.

File: src/screens/about/components/showcase-app.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const ShowcaseApp = (props) => (
```
Comment (ebrillhart) [inline src/screens/about/components/showcase-app.js]: can you convert this to a React.Component component please? just to keep things consistent

**Comment by ebrillhart** [comment]
@rawrmonstar overall this looks good to me if you make that conversion! I'll also let @chrisbolin weigh in on the code side of things. I'll update content and @paulathevalley can update design when she's back next week. nice work!

**Comment by chrisbolin** [comment]
Code so far looks good to me! (and it built just fine on my machine)

Might be a good idea to merge in the master, as this branch has diverged. It's always nice to stay current with `master`, then you can fix any broken things as they happen.

**Comment by ebrillhart** [comment]
@paulathevalley I've updated the content on the about page/showcase. LMK if you need anything else from me on this branch!

**Comment by paulathevalley** [comment]
@ebrillhart :+1: Thank you! I might design in a new branch and PR to merge it back into this one. So, when this PR gets merged, it'll deploy and everything will be beautiful.

**Comment by paulathevalley** [comment]
@ebrillhart Ready for re-review/Good to merge?

**Comment by ebrillhart** [comment]
LGTM!

**Pull request merged**